### PR TITLE
fix(DB/Halls of Stone) Trash inside dungeon now drops [Relic of Ulduar]

### PR DESCRIPTION
**Pull request by blinkysc**
npc 27966 doesn't show any drop chance in wowhead. All chances and min/max based on data from current Wotlk Wowhead https://www.wowhead.com/wotlk

27960
27961
27962
27963
27964
27965
~~27966~~
27969
27970
27971
27972

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/21704

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .go c id [id above] 
2. kill and see if loot
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by blinkysc** [comment]
@Kitzunu have PTSD deleting from *_template is commit https://github.com/azerothcore/azerothcore-wotlk/pull/21710/commits/13820abe3c680b3f4c402530e74bf2b2268bca30 appropriate for codestyle? Or is DELETE from creature_loot_template in this instance fine?

File: data/sql/updates/pending_db_world/rev_1741995038650745837.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+(27972, 42780, 0, 19, 0, 1, 0, 1, 3, 'Lightning Construct - Relic of Ulduar'),
+ON DUPLICATE KEY UPDATE 
+    Reference = VALUES(Reference),
+    Chance = VALUES(Chance),
+    QuestRequired = VALUES(QuestRequired),
+    LootMode = VALUES(LootMode),
+    GroupId = VALUES(GroupId),
+    MinCount = VALUES(MinCount),
+    MaxCount = VALUES(MaxCount),
+    Comment = VALUES(Comment);
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1741995038650745837.sql:23]: we don't do this because it's intransparent. please make all changes explicit with `UPDATE` or `DELETE` / `INSERT`.

Comment (sudlud) [reply data/sql/updates/pending_db_world/rev_1741995038650745837.sql:23]: also if there is an `INSERT`, there must be a `DELETE` before.

**Review by sudlud — CHANGES_REQUESTED** [review]

**Review by sudlud — APPROVED** [review]
